### ISSUE TITLE
Empty lines between sequential .proc or .endproc. Empty line should be inserted only after last .endproc

### ISSUE THREAD

**Issue by An-S** [An-S/sf65#5]
label5:
            .ENDSCOPE

        .ENDSCOPE

anoversizedlabel

**Comment by An-S** [comment]
The parser recognizes sequences of .proc's or .endproc's etc correctly. However, the formatter outputs another number of spaces in an empty line then in the original which was misleading.
Therefore i opened another issue for this
